portstyle.css

```diff
@@ -206,5 +202,16 @@ footer {
 	}
```

Comment:
you're using grid properties in a flex-box?

Reply:
It's not really showing me the part you're questioning. Is this with regards to the media query for .aboutMe?